gramps/gui/plug/_windows.py

```diff
@@ -431,6 +431,9 @@ def __init__(self, dbstate, uistate, track):
         hbox.pack_start(self.addon_combo, False, False, 0)
 
         self.projects = config.get("behavior.addons-projects")
+        # strip the incoming URLs to avoid any leading or trailing whitespace
+        for project in self.projects:
+            project[1] = project[1].strip()
```

Comment:
I would prefer to ensure that urls were stripped of whitespace before storing in the preferences.

Reply:
With the current approach all URLs are stripped on the way in to gramps (either from the preference - this code, or when a new URL is entered by the user - the other change in this PR). Therefore I don't think there is any need to strip when storing in the preferences. As a by product the URLs are whitespace free during the lifetime of the addon manager dialog.
Stripping URLs when reading the preference covers the scenario of an upgraded 5.2 preference file already containing whitespace on the URLs.